src/sagas/favorites.js

```diff
@@ -0,0 +1,59 @@
+import { takeEvery, put, select } from 'redux-saga/effects';
+import favoriteActions from '../actions/favorites';
+import Utils from '../utils/Utils';
+import selectors from '../selectors';
+
+export function* getFavoritesList() {
+    const localFavs = JSON.parse(localStorage.getItem('favList')) || [];
+    yield put(favoriteActions.creators.getFavoritesListComplete(localFavs));
+}
+
+export function* addFavorite(fav) {
+    const localFavs = JSON.parse(localStorage.getItem('favList')) || [];
+    localFavs.push(fav.payload);
+    const newLocal = JSON.stringify(localFavs);
+    localStorage.setItem('favList', newLocal);
+    yield put(favoriteActions.creators.addFavoriteComplete(localFavs));
+}
+
+export function* removeFavorite(id) {
+    const localFavs = JSON.parse(localStorage.getItem('favList')) || [];
```

Comment:
This `const localFavs = JSON.parse(localStorage.getItem('favList')) || [];` line is repeated a couple of times, a function would be nice in case you need a change.